staging/src/k8s.io/apimachinery/pkg/runtime/serializer/cbor/internal/modes/roundtrip_test.go

```diff
@@ -32,6 +32,11 @@ func TestRoundtrip(t *testing.T) {
 		dec cbor.DecMode
 	}
 
+	var absentSlice []interface{}
+	var absentMap map[string]interface{}
```

Comment:
The zero value for these makes them identical to the expressions `[]interface{}(nil)` and `map[string]interface{}(nil)`, so the uninitialized cases can be dropped without losing coverage.

Reply:
Removed these cases. Thanks!